torch/ao/quantization/experimental/APoT_tensor.py

```diff
@@ -2,13 +2,13 @@
 from torch.ao.quantization.experimental.quantizer import APoTQuantizer
 
 # class to store APoT quantized tensor
-class TensorAPoT(torch.Tensor):
+class TensorAPoT():
     quantizer: APoTQuantizer
-    data_reduced_precision_repr: torch.Tensor
-    data_int_repr: torch.Tensor
+    data: torch.Tensor
 
     def __init__(self, quantizer):
-        raise NotImplementedError
+        self.quantizer = quantizer
+        self.data = quantizer.data
```

Comment:
feels like the data should be stored in the tensor rather than in the quantizer, i would probably refactor the quantizer to take the data as an argument for this PR

Reply:
fixed!